package-lock.json

```diff
@@ -1581,6 +1581,14 @@
       "resolved": "https://registry.npmjs.org/cluster-key-slot/-/cluster-key-slot-1.1.0.tgz",
       "integrity": "sha512-2Nii8p3RwAPiFwsnZvukotvow2rIHM+yQ6ZcBXGHdniadkYGZYiGmkHJIbZPIV9nfv7m/U1IPMVVcAhoWFeklw=="
     },
+    "coffee-script": {
+      "version": "1.8.0",
+      "resolved": "https://registry.npmjs.org/coffee-script/-/coffee-script-1.8.0.tgz",
+      "integrity": "sha1-nJ8dK0pSoADe0Vtll5FwNkgmPB0=",
```

Comment:
Side note: The deleted entry in the redis package had a strong sha :shrug:

Reply:
Good catch! I tried to scrub the delta to keep the strong shas. I think I'll alter this one so that we keep the strong sha, even though the specification moved.

Reply:
I do not think it's worth the effort -- I can tackle it as part of https://github.com/overleaf/issues/issues/3252

Reply:
cc663a3 worked locally `$ bin/run --no-deps document-updater npm ci` :+1: